docs/04-interfaces/03-prci/03-usage-without-pce.mdx

```diff
@@ -0,0 +1,16 @@
+---
+title: Usage Without PCE
+sidebar_position: 3
+tags: [PRCI, web-components]
+---
+
+The PRCI component can be used without an associated Privacy Computation Engine. The example below exposes a PRCI element on a static HTML page.
+
+```html
+<body>
+  <bldn-priv-request></bldn-priv-request>
+  <script src="https://cdn.jsdelivr.net/npm/@blindnet/prci@0.12.3/dist/index.all.min.js" type="module"></script>
+</body>
+```
+
+When a request is filled out and submitted, the PRCI will emit the [`bldn-priv-request:request-created`](./events#bldn-priv-requestrequest-created) event. You may catch this event and handle it in your system.
```

Comment:
@jboileau99 Can you give a example (code block) of how to handle the event? And add it to the quickstart repo.

Reply:
added in [075bf90](https://github.com/blindnet-io/blindnet.dev/pull/80/commits/075bf90fac72701384e2fd7b4a1db3b25fa54ca3)